src/components/albums/albums.service.js

```diff
@@ -0,0 +1,20 @@
+export class AlbumsService {
+  constructor($q, $http) {
+    'ngInject;'
+    this.$q = $q;
+    this.$http = $http;
+    this.baseUrl = '/search?q=';
```

Comment:
This should not be here, also, for query string, use the `params` prop inside the configs of http provider
https://docs.angularjs.org/api/ng/service/$http#get